tasties/urls.py

```diff
@@ -16,6 +16,8 @@
 from django.contrib import admin
 from django.urls import path
 from tasties_app import views
+from . import settings
```

Comment:
You should consider using absolute imports instead of relative
`from tasties_app`

Reply:
Corrected.